mkdocs.yml

```diff
@@ -13,6 +13,7 @@ nav:
       - Get started with AWS:
           - Overview: aws/index.md
           - AWS Landing Zone overview: aws/get-started-with-aws/bc-govs-aws-landing-zone-overview.md
+          - Security guardrails: aws/get-started-with-aws/security-guardrails.md
```

Comment:
Should the navigation show "for AWS" (or equivalent), since it's pointing to the AWS documentation? Will there be an Azure version as well?

Reply:
I imagine we will have something similar for Azure. This has been there for a while but we never added it to the mkdocs (oops)

Reply:
Changed to  "AWS Security and compliance guardrails"